DAY 6:

amaliahern opened the pull request:
.the result of the fizzbuzz kata is wrong but I learnt about shovel operator. Victor is my Ruby angel. :-)

Commit:
DAY 6: the result of the fizzbuzz kata is wrong but I learnt about shovel operator. Victor is my Ruby angel. :-)

kotp commented on 2_adventures/001/amaliahern/06/fizz_buzz.rb:
```diff
@@ -0,0 +1,16 @@
+def fizzbuzz(number)
+  result = ""
+  if(number % 3 == 0)
+    result = DIVISIBLE_BY_3 
```
You will want to shovel the answer here instead of replacing the empty string.

kotp commented on 2_adventures/001/amaliahern/06/fizz_buzz.rb:
```diff
@@ -0,0 +1,16 @@
+  elsif(number % 5 == 0)
+    result << DIVISIBLE_BY_5 
+  elsif
+    result = number
```
You have an if, then elsif.  Walk through this logic and you will see where it is going wrong.

kotp commented on 2_adventures/001/amaliahern/06/fizz_buzz.rb:
```diff
@@ -0,0 +1,16 @@
+  elsif
+    result = number
+  end
+  result
```
You can do this better without a branched conditional statement.  Tell your value to be shovelled on the result if some condition.  And do that as many times as you need to.

Then return what you want based on the result of the repetitive "filter".  Think about that toy that has the wholes that are smaller and smaller until finally, you have filtered everything but the answer.

kotp commented:
I should add, if anyone wants to pair program, they only need to chat me in the Gitter.im which is a great way to get my attention at the moment.

Commit:
DAY 7
I did the suggestion of Viktor and I get Green. Hurray! Could I improve more fizz_buzz kata? Sure.

Commit:
README.md file updated

amaliahern commented:
Hi Viktor, 
I will be delighted to pairing with you but I think the time zone is a huge gap. I am in Spain.

kotp commented:
It very well could be a large problem.  I usually work in India Time Zone.  So that may indeed be prohibitive.  I usually stop pairing and doing these things near the 5pm IST hour.

patmaddox commented:
Go you!
